CONTRIBUTING.md

```diff
@@ -0,0 +1,133 @@
+# Contributing
```

Comment:
@abaicus let's add also a How to release section here,https://github.com/Codeinwp/otter-blocks/blob/master/CONTRIBUTING.md